symphony/bdk/core/auth/jwt_helper.py

```diff
@@ -1,25 +1,73 @@
+"""Module to help with jwt handling.
+"""
 import datetime
-from jose import jwt
 
-DEFAULT_EXPIRATION = 5 * 58
+from cryptography.hazmat.primitives.serialization import Encoding, PublicFormat
+from cryptography.x509 import load_pem_x509_certificate
 
+from jose import jwt, JOSEError
 
-def create_signed_jwt(private_key_config, username, expiration=None):
+from symphony.bdk.core.auth.exception import AuthInitializationError
+from symphony.bdk.core.config.model.bdk_rsa_key_config import BdkRsaKeyConfig
+
+RS512 = "RS512"
+
+DEFAULT_EXPIRATION = 290
```

Comment:
Maybe put in here again the explanation on why this value?
or write 5 * 60 - 10? that would help understand it is 4min 50s